app/js/components/mediaPlayer.js

```diff
@@ -112,8 +114,53 @@ var MediaPlayer = React.createClass({
     Actions.seekTrack(time)
   },
 
-  toggleTimeLeft: function() {
-    this.setState({ 'timeLeft' : !this.state.timeLeft })
+  setVolume: function (event) {
+    var volume = event.target.value
+
+    this.setState({
+      lastVolume: this.state.audio.volume,
+      isMuted: false
+    })
+    Actions.setVolume(volume)
+  },
+
+  setVolumeWithWheel: function (event) {
+    var volume = this.state.audio.volume;
+    var newVolume = volume +
+      Math.sign(
+        Math.abs(event.deltaX) >= Math.abs(event.deltaY) ? event.deltaX
+          : -event.deltaY
+      ) * 0.025;
+
+    // make sure new volume level is in range [0,1]
+    if (newVolume > 1)
+      newVolume = 1
+    else if (newVolume < 0) {
+      newVolume = 0
+    }
+
+    this.setState({
+      lastVolume: volume,
```

Comment:
Let's make sure we have a fixed length.
Using `volume.toFixed(2)` will truncate the volume level to 2 numbers at the most.

e.g. `0.45555555` will become `0.45`